demo.py

```diff
@@ -237,6 +237,7 @@ def prompt_progress_callback(percent):
         model_kit,
         prompt_tokens,
         images_b64=images_base64,
+        max_image_size=1000,
```

Comment:
```suggestion
        max_image_size=1024,
```

to match app default

Reply:
Updated to `1024`